lib/index.js

```diff
@@ -1,167 +1,150 @@
+        };
+        return __assign({}, result, (_a = {}, _a[operationId] = gType, _a));
+    }, {});
+};
+var schemaFromEndpoints = function (endpoints, proxyUrl, headers) {
+    var gqlTypes = {};
+    var queryFields = getFields(endpoints, false, gqlTypes, proxyUrl, headers);
+    if (!Object.keys(queryFields).length) {
+        throw new Error('Did not find any GET endpoints');
+    }
+    var rootType = new graphql_1.GraphQLObjectType({
+        name: 'Query',
+        fields: queryFields,
     });
-  }
-
-  return new _graphql.GraphQLSchema(graphQLSchema);
+    var graphQLSchema = {
+        query: rootType,
+    };
+    var mutationFields = getFields(endpoints, true, gqlTypes, proxyUrl, headers);
+    if (Object.keys(mutationFields).length) {
+        graphQLSchema.mutation = new graphql_1.GraphQLObjectType({
+            name: 'Mutation',
+            fields: mutationFields,
+        });
+    }
+    return new graphql_1.GraphQLSchema(graphQLSchema);
 };
-
-var build = function () {
-  var _ref2 = (0, _asyncToGenerator3.default)( /*#__PURE__*/_regenerator2.default.mark(function _callee2(swaggerPath) {
-    var proxyUrl = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : null;
-    var headers = arguments[2];
+var build = function (swaggerPath, proxyUrl, headers) { return __awaiter(_this, void 0, void 0, function () {
     var swaggerSchema, refs, endpoints, schema;
-    return _regenerator2.default.wrap(function _callee2$(_context2) {
-      while (1) {
-        switch (_context2.prev = _context2.next) {
-          case 0:
-            _context2.next = 2;
-            return (0, _swagger.loadSchema)(swaggerPath);
-
-          case 2:
-            swaggerSchema = _context2.sent;
-            _context2.next = 5;
-            return (0, _swagger.loadRefs)(swaggerPath);
-
-          case 5:
-            refs = _context2.sent;
-            endpoints = (0, _swagger.getAllEndPoints)(swaggerSchema, refs);
-            schema = schemaFromEndpoints(endpoints, proxyUrl, headers);
-            return _context2.abrupt('return', schema);
-
-          case 9:
-          case 'end':
-            return _context2.stop();
+    return __generator(this, function (_a) {
+        switch (_a.label) {
+            case 0: return [4 /*yield*/, swagger_1.loadSchema(swaggerPath)];
+            case 1:
+                swaggerSchema = _a.sent();
+                return [4 /*yield*/, swagger_1.loadRefs(swaggerPath)];
+            case 2:
+                refs = _a.sent();
+                endpoints = swagger_1.getAllEndPoints(swaggerSchema, refs);
+                schema = schemaFromEndpoints(endpoints, proxyUrl, headers);
+                return [2 /*return*/, schema];
         }
-      }
-    }, _callee2, undefined);
-  }));
-
-  return function build(_x5) {
-    return _ref2.apply(this, arguments);
-  };
-}();
-
-exports.default = build;
-module.exports = exports.default;
\ No newline at end of file
+    });
+}); };
+module.exports = build;
```

Comment:
adding a default export didn't work yet, maybe we have to try harder.

I didn't try `build.default = build` yet.

Reply:
fixed it